scripts/validation/user-create.avsc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@smoya 

Added this file and changed the references in the spec. Commit details: f814d826

It's validating perfectly now with no errors whatsoever.